tech/languages/python/pipenv.md

```diff
@@ -301,59 +292,36 @@ pipenv
 
 # Create a new project using Python 3:
 pipenv --three
-
-# Install a package:
-pipenv install package_name
-
-# Install all the dependencies for a project:
-pipenv install
-
-# Install all the dependencies for a project (including dev packages):
-pipenv install --dev
-
-# Uninstall a package:
-pipenv uninstall package_name
-
-# Start a shell within the created virtual environment:
-pipenv shell
-
-# Generate a `requirements.txt` (list of dependencies) for a project:
-pipenv lock --requirements
+......
 ```
 
 You can also run it directly using `pipenv run` without activating the
 shell.
 
-```$ pipenv run python http-request.py http://cheat.sh/pipenv
-# pipenv
-# Simple and unified Python development workflow.
-# Manages packages and the virtual environment for a project.
-# More information: <https://pypi.org/project/pipenv>.
-
-# Create a new project:
-pipenv
-
-# Create a new project using Python 3:
-pipenv --three
+```
+$ pipenv run python http-request.py http://cheat.sh/pipenv
+......
```

Comment:
```suggestion
......
```

No need for `.....` if there's no output.

Reply:
Actually here the output is exactly the same as the output of previous command. (As we are running the same script.)
It's kind of redundant, so did not put it there.